lib/datasource/remote/firebase/firebase_remote_config.dart

```diff
@@ -13,16 +13,26 @@ const String _dhoExplorerUrlKey = 'dho_explore_url';
 const String _defaultEndPointUrlKey = 'default_end_point';
 const String _defaultV2EndPointUrlKey = 'default_v2_end_point';
 
-const String _eosEndpoints = '[ { "url": "https://api.telosfoundation.io", "isDefault": true } ]';
```

Comment:
Changes that dont belong here are  because you are pointing to another branch instead of master, and you probably have changes here from that other branch that are not in master yet. 

This Should be resolved when you merge your other branch into master and you point this branch into master